build/bin/vlan-filtering

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+IFS=$'\n'
+sed_expression="^.*: (.*) state .* : .*master (.*) state.*$"
+vlan_range=1-4094
+for port in $(bridge link show | grep -v @); do
+   outbound_name=$(echo $port | sed -r "s/$sed_expression/\1/")
+   bridge_name=$(echo $port | sed -r "s/$sed_expression/\2/")
+   bridges=$(printf $bridge_name\\n$bridges | uniq)
```

Comment:
a safer way to list all host bridges:
```
ip -o link show type bridge
```

Reply:
I am setting vlan filtering config to bridges that have a port to outbound interface filtering ports with "grep -v @".
ip -o link show type bridge give you all bridges even if they don't have a port to an outbound interface.

Reply:
I think that you should set `vlan_filtering=1` even on floating nic-less bridges.

Reply:
I am setting it now at all bridges from desiredState so I cover that, I am going to mark this as resolved.